cmd/utils/miscellaneous.go

```diff
@@ -32,6 +33,7 @@ func CreateRecorder(kubeClient k8s.Interface, eventSourceName string) record.Eve
 	scheme := scheme.Scheme
 
 	coreinstall.Install(scheme)
+	seedmanagementinstall.Install(scheme)
```

Comment:
When you're already on this file, can you please construct a new scheme here? Using `scheme.Scheme` directly can cause concurrency issues.
```suggestion
	scheme := runtime.NewScheme()

	utilruntime.Must(kubernetesscheme.AddToScheme(scheme))
	coreinstall.Install(scheme)
	seedmanagementinstall.Install(scheme)
```

Reply:
OK